google/ggrc-core
# Fix Network Zone for Systems

# Issue description

Network Zone of System/Process are not synced between GGRC and GGRCQ

# Steps to test the changes

1. Create System/Process on GGRCQ
2. Set Network Zone
3. Sync and check on GRGC

# Solution description

During the building of Network Zone on GGRC attribute was considered as an object of Option type. However, it was only a title of such object. This fix includes loading an object from known title for further processing.

# Sanity checklist


# PR Review checklist
